presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -297,6 +297,26 @@ public static Slice spookyHashV2_64(@SqlType(StandardTypes.VARBINARY) Slice slic
         return hash;
     }
 
+    @Description("compute Thomas Wang downscaling hash function")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice twang64From32(@SqlType(StandardTypes.VARBINARY) Slice slice)
```

Comment:
There are several references that talk about various Thomas Wang hash functions, but I couldn't find one that talks about this variant. Can you please point me to a reference for this function?